src/libs/API.js

```diff
@@ -481,6 +482,23 @@ function SetPassword(parameters) {
     return request(commandName, parameters);
 }
 
+/**
+ * Display a error message in the UI when the API returns an authentication error.
+ *
+ * @param {String} error
+ * @returns {String}
+ */
+function GetSignInErrorMessage(error) {
```

Comment:
Ah what I meant was that you should probably place this in `Authenticate`. I also think you don't need a separate authenticationErrorMessages file either. It's overkill for what we're trying to achieve IMO

Reply:
What I understood where you mentioned place in `Authenticate`. This means in `src\libs\actions\Session.js` file. So, This function `GetSignInErrorMessage ` should be in this `src\libs\actions\Session.js` file, and no need separate file for `authenticationErrorMessages`. If you confirm me so I can proceed the changes.

Reply:
@yuwenmemon Could you please confirm so I can push and close it?